library/sequencer/test/sequence_test.rb

```diff
@@ -65,5 +71,18 @@ def second
 
       expect(subject.from_methods(seq)).to eq(wanted)
     end
+
+    it "does not confuse skip_stack across classes" do
+      subj = UnrelatedTestSequence.new
+      seq = {
+        "ws_start" => "first",
+        "first"    => { next: :next }
+      }
+      wanted = {
+        "first" => [subj.method(:first), true]
+      }
+
+      expect(subj.from_methods(seq)).to eq(wanted)
```

Comment:
I do not get this test. I expect it should test if TestSequence do not skip `"first"`, not? Here it test that class that explicitly skip it really skip it, which is not what "it" description mention